### DISCUSSION TITLE
browser biometrics option missing in chrome extension?

### DISCUSSION THREAD

**Discussion by zany130** [quexten/goldwarden#198] [Q&A] [answered]
EDIT:
didn't see updated wiki page
https://github.com/quexten/goldwarden/wiki/Browser-Biometric-Approval

It's not really a Goldwarden issue or question, but...

Has anyone noticed there no longer seems to be an option for browser biometrics in the official Chrome extension?

used to be where I drew the red line? 
![image](https://github.com/quexten/goldwarden/assets/25832539/e88982ce-dad3-490c-8282-c4b1b84f97d8)

**Comment by quexten** [comment, marked as answer]
Yeah, as written in the wiki, upstream decided to remove the option on linux since the official desktop app does not support it (yet). Need to see if I can discuss what's required to get my PR to get support for the desktop app merged.
